setup.cfg

```diff
@@ -0,0 +1,38 @@
+[metadata]
+name = openfe
+version = 0.0.1.dev0
+description = 
+long_description = file: README.md
+long_description_type = text/markdown
+#author =
+#author_email =
+license = MIT
+license-file = LICENSE
+url = https://github.com/OpenFreeEnergy/openfe
+classifiers =
+    Development Status :: 1 - Planning
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: MIT License
+    Operating System :: POSIX
+    Operating System :: Microsoft :: Windows
+    Programming Language :: Python :: 3
+    Topic :: Scientific/Engineering :: Bio-Informatics
+    Topic :: Scientific/Engineering :: Chemistry
+
+[options]
+python_requires = >= 3.8
+install_requires =
+    numpy
+    networkx
+    rdkit-pypi
```

Comment:
It's not clear to me that `rdkit-pypi` is actually a product of the rdkit developers (indeed the fact that it claims to support py3.10 before there being one on conda-forge seems a bit suspicious).

Reply:
Yeah conda-forge is the official source afaik. We're going to have to have conda deps at some point I think